M2/Macaulay2/d/common.d

```diff
@@ -110,14 +110,6 @@ export setup(e:SymbolClosure,fn:ternop):void := (
 export setup(e:SymbolClosure,fn:multop):void := (
      multopNameList = multopNameListCell(fn,e.symbol,multopNameList);
      );
-export setupfun(name:string,fun:unop):Symbol := (
```

Comment:
What's the rationale for removing this?

Reply:
It's not called anywhere, and I'm not sure it even works.  While I was playing around with these changes, I unsuccessfully tried using it to define `TEST` as a keyword.

Reply:
Could this be used for something like `not`?

Reply:
I played around with this a bit more, and keywords created with this function end up being left associative instead of right associative like we'd probably expect.  So if we were to use it to define `not`, something like `not f x` would be parsed as `(not f) x` instead of `not (f x)`.